Patches/CustomDataLoader/CreateCardClonesPostfix.cs

```diff
@@ -14,12 +14,12 @@ public class CreateCardClonesPostfix
 {
     private const string ItemDirectoryPath = @"BepInEx\plugins\characters\";
     // public static string[] classes = new string[100];
-    private static List<string> classes = new List<string>();
+    private static Dictionary<string, SubClassData> classes = new Dictionary<string, SubClassData>();
     
     [HarmonyPostfix]
     static void LoadCharacterData(Dictionary<string, SubClassData> ____SubClass)
     {
-        classes = ____SubClass.Keys.ToList();
+        classes = ____SubClass;

```

Comment:
can we either do this at the end or only use this accessor, since other places use `classes` or `___SubClass`